packages/@ionic/cli-utils/src/lib/project/angular/build.ts

```diff
@@ -18,73 +44,45 @@ export class BuildRunner extends BaseBuildRunner<AngularBuildOptions> {
       description: `
 ${chalk.green('ionic build')} uses the Angular CLI. Use ${chalk.green('ng build --help')} to list all Angular CLI options for building your app. See the ${chalk.green('ng build')} docs${chalk.cyan('[1]')} for explanations. Options not listed below are considered advanced and can be passed to the ${chalk.green('ng')} CLI using the ${chalk.green('--')} separator after the Ionic CLI arguments. See the examples.
 
-${chalk.cyan('[1]')}: ${chalk.bold('https://github.com/angular/angular-cli/wiki/build#ng-build')}`,
-      options: [
-        {
-          name: 'dev',
-          summary: `Sets the build target to ${chalk.green('development')}`,
-          type: Boolean,
-          hint: 'ng',
-        },
-        {
-          name: 'prod',
-          summary: `Sets the build target to ${chalk.green('production')}`,
-          type: Boolean,
-          hint: 'ng',
-        },
-        {
-          name: 'target',
-          summary: 'Set the build target to a custom value',
-          aliases: ['t'],
-          groups: [OptionGroup.Advanced],
-          hint: 'ng',
-        },
-        {
-          name: 'environment',
-          summary: 'Set the build environment to a custom value',
-          aliases: ['e'],
-          groups: [OptionGroup.Advanced],
-          hint: 'ng',
-        },
-      ],
+${chalk.cyan('[1]')}: ${chalk.bold('https://github.com/angular/angular-cli/wiki/build')}`,
+      options: NG_BUILD_OPTIONS,
     };
   }
 
   createOptionsFromCommandLine(inputs: CommandLineInputs, options: CommandLineOptions): AngularBuildOptions {
     const baseOptions = super.createBaseOptionsFromCommandLine(inputs, options);
-    let target = options['target'] ? String(options['target']) : undefined;
-    const environment = options['environment'] ? String(options['environment']) : undefined;
-
-    if (!target) {
-      if (options['dev']) {
-        target = 'development';
-      } else if (options['prod']) {
-        target = 'production';
-      }
-    }
+    const ngOptions = NG_BUILD_OPTIONS
```

Comment:
We do lose some type info here by doing this, but that's alright if there's tests, which there are